packages/fxa-auth-server/lib/routes/recovery-key.js

```diff
@@ -296,8 +296,10 @@ module.exports = (
       options: {
         ...RECOVERY_KEY_DOCS.RECOVERYKEY_EXISTS_POST,
         auth: {
-          mode: 'optional',
```

Comment:
auth is no longer optional for this endpoint.

Reply:
Should also update the swagger docs to reflect the change?